tests/unit/utils/formatUserDetails.test.ts

```diff
@@ -0,0 +1,23 @@
+import { formatUserDetails } from "../../../src/utils/formatUserDetails";
+
+describe("formatUserDetails function", () => {
+  it("should format user details correctly", () => {
+    const user = JSON.stringify({
+      user: {
+        first_name: "John",
+        last_name: "Doe",
+        discordJoinedAt: "2023-09-13T09:30:00.000Z",
+        state: "Active",
+      },
+    });
+
+    const formattedDetails = formatUserDetails(user).trim();
```

Comment:
Nit: why used trim?

Reply:
In order to remove any white spaces before and after the string so that the expected test case matches with the resulting data.